src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -38,13 +38,44 @@
         .bcs-activity-feed-task-new,
         .bcs-activity-feed-comment,
         .bcs-activity-feed-app-activity {
-            margin-top: 20px;
+            position: relative;
+            padding-top: 20px;
             padding-bottom: 20px;
             border-bottom: 1px solid $bdl-gray-10;
 
             &:first-child .bcs-comment {
                 margin-top: 0;
             }
+
+            &.bcs-is-focused::before {
+                position: absolute;
+                top: 0;
+                right: 0;
+                bottom: 0;
+                left: 0;
+                margin-right: -25px;
+                margin-left: -25px;
+                background-color: transparent;
+                animation: fade-out 6s ease-out;
+                content: '';
+
+                @keyframes fade-out {
```

Comment:
We should use `$bdl-box-blue-05` here, since we don't want the focus color to change if the background changes, right?